src/components/MainImage.js

```diff
@@ -1,15 +1,18 @@
-import React, { Component } from 'react'
-import Illustration from '../images/illustration-mockups.svg'
-export class MainImage extends Component {
-  render() {
+import React from 'react';
+import Illustration from '../images/illustration-mockups.svg';
+import '../css/mainImage.css';
+
+function MainImage ({headerIcon, header}) {
     return (
       <div className="illustration">
         <div className="header">
-         <h3 className="fadeIn-header"><span><i className="fa fa-comments"></i></span> Huddle</h3>
+          <h3 className="fade-in-header"><span><i className={headerIcon}></i> {header}</span>
+          </h3>
         </div>
-        <img className="fadeIn-image" src={Illustration} alt="Main-img"/>
+        <img className="fade-in-image" src={Illustration} alt="Main-img"/>
```

Comment:
I would suggest leaving the alt attribute empty. If you think about it, this illustration isn't important content and needed for the user to understand what the site or surrounding information means. It's purely design fluff to accommodate text.

so use `alt=""` instead of what you have currently. You still have to use the empty quotations otherwise a screenreader won't know what to do and read the file name.